src/components/PageMetadata/PageMetadata.tsx

```diff
@@ -71,35 +75,35 @@ const Authors = styled(({ className, path,  repoType, repo, contentRootPath, loc
     // Flatting to array of author and sort by commits number descending
     const sortedByCommitsCountAuthors = Object.entries(uniqueAuthors)
       .map(([, author]) => author)
-      .sort((a, b) => (a.commitsCount < b.commitsCount) ? 1 : -1)
+      .sort((a, b) => (a.commitsCount < b.commitsCount) ? 1 : -1);
 
     // Set to React state
-    setAuthorList([...sortedByCommitsCountAuthors]);
+    setAuthorList(sortedByCommitsCountAuthors);
 
     // Last update
     const lastCommitDate = response.data[0].commit.author.date;
-    setLastUpdate(format(new Date(lastCommitDate),"MMM dd, yyyy"));
+    setLastUpdate(format(new Date(lastCommitDate), "MMM dd, yyyy"));
```

Comment:
I know we should have a better auto code formatting but could you please be consistency use single quote. ^^
Thank you.

Reply:
OK